rplugin/python3/translate.py

```diff
@@ -2,7 +2,49 @@
 import tempfile
 import pynvim
 from googletrans import Translator
-
+from enum import Enum
+
+class DisplayType(Enum):
+    STATUS = 0
+    POPUP = 1
+
+class CTERMColours(Enum):
+    BLACK = 0
+    DARKBLUE = 1
+    DARKGREEN = 2
+    DARKCYAN = 3
+    DARKRED = 4
+    DARKMAGENTA = 5
+    BROWN = 6
+    GREY = 7
+    DARKGREY = 8
+
```

Comment:
Many thanks for this color Enum.  It is helpful to use the color.  It just a question not a request, Why there is an empty line after these 9 elements? Maybe you want to leave this empty line after 8 elements?

Reply:
No problem, I am glad it is helpful! I added the space to indicate the difference in the item ordering between the two enums. Would you like the space moved/removed?

Reply:
Got it, please keep it.  Thanks for the details.